docs/examples/embed.js

```diff
@@ -0,0 +1,65 @@
+/*
+	An example of how you can send embeds
+*/
+
+// Extract the required objects from the discord.js module
```

Comment:
i would call it `Extract the required classes` because they are more than normal objects tho